core/src/transaction_meta.rs

```diff
@@ -32,12 +34,14 @@ impl TransactionMeta {
     pub fn new(
         block_number: u64,
         epoch_number: u64,
+        block_hash: H256,
```

Comment:
```suggestion
    pub fn new(block_info: BlockInfo, outputs_count: usize, all_dead: bool)
```

Reply:
Yes. I did, but then I deleted it after talking with @yangby-cryptape . @yangby-cryptape will refactor this part later. 
So this PR only focus its goal without any other refactorings.